packages/parcel-transformer-inject-env/src/index.ts

```diff
@@ -0,0 +1,18 @@
+import { Transformer } from "@parcel/plugin"
+
+const envRegex = /\$(PLASMO_PUBLIC_[(\w+)]+)/gm
```

Comment:
This matches stuff like

`$PLASMO_PUBLIC_ASDF()ASDFSADF(+FDS` is this intended? 

I think the `[` `]` were placed there accidentally

Reply:
rip you are right, I was playing around with grouping and used the wrong one, then kept it there lol